gnome_control_center-started-20170831.json

```diff
@@ -16,7 +16,8 @@
     }
   ],
   "tags": [
-    "gnome-control-center-started"
+    "gnome-control-center-started",
+    "gnome-control-center-new-layout"

```

Comment:
I think 'new' is not good word here , because you will get into a trouble when newer will appear :) 
suggest to change it to version number ? "gnome-control-center-3-26-layout" ?

Reply:
and in 6 month you want to tag GNOME 3.28 control-center with a 3.26-layout needle? That will be even more confusing imho - and chances are low that the entire shell will be switched.

Other/better alternatives?

Reply:
no better alternatives , so you can leave it as it is